build/ci/windows/sign.bat

```diff
@@ -0,0 +1,66 @@
+@echo off
+ECHO "MuseScore sign"
+
+SET CERT_ENCRYPT_SECRET=""
+SET CERT_PASSWORD=""
+SET FILES_DIR=""
+SET FILE_PATH=""
+SET FILE_NAME=""
+
+SET SIGNTOOL="C:\Program Files (x86)\Windows Kits\10\bin\x64\signtool.exe"
+SET SIGNCERT="build\ci\windows\resources\musescore.p12"
+
+::SET TIMESERVER1="http://timestamp.globalsign.com/scripts/timstamp.dll"
+SET TIMESERVER1="http://timestamp.verisign.com/scripts/timstamp.dll"
+SET TIMESERVER2="http://timestamp.sectigo.com"
```

Comment:
This one redirects to an https page, https://sectigo.com/resource-library/time-stamping-server